app/src/main/res/layout-land/promoted_story_list.xml

```diff
@@ -63,6 +63,6 @@
       android:paddingStart="@dimen/home_padding_start"
       android:paddingEnd="@{viewModel.endPadding}"
       app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-      app:list="@{viewModel.promotedStoryList}" />
+      app:dataList="@{viewModel.promotedStoryList}" />
```

Comment:
To avoid this being too generic/close to the generic binding adapter, suggest calling the binding 'promotedStoryList' to keep it context-specific.

Reply:
Done